ui/app/templates/components/secret-list-header.hbs

```diff
@@ -0,0 +1,108 @@
+{{#with (options-for-backend model.type) as |options|}}
+  {{#page-header as |p|}}
+    {{#p.top}}
+      {{#key-value-header
+        baseKey=baseKey
+        path="vault.cluster.secrets.backend.list"
+        root=backendCrumb
+      }}
+        <li>
+          <span class="sep">&#x0002f;</span>
+          <a href={{href-to "vault.cluster.secrets"}}>
+            secrets
+          </a>
+        </li>
+      {{/key-value-header}}
+    {{/p.top}}
+    {{#p.levelLeft}}
+      <h1 class="title is-3">
+        {{model.id}}
+        <span class="tag is-outlined is-inverted has-text-grey-dark is-font-mono">
+          {{or options.displayName (capitalize model.type)}}
+        </span>
+        {{#if (eq model.options.version 2)}}
+          <span class="has-text-grey-dark has-text-weight-normal is-size-6">
+            Version 2
+          </span>
+        {{/if}}
+      </h1>
+    {{/p.levelLeft}}
+    {{#p.levelRight}}
+       {{#unless (or isCertTab isConfigure)}}
+          <div class="control">
+            {{#secret-link
+              mode="create"
+              secret=(or baseKey.id '')
+              queryParams=(query-params initialKey='')
+              class="button has-icon-right is-ghost is-compact"
+              data-test-secret-create=true
+            }}
+              {{options.create}}
+              {{i-con glyph="chevron-right" size=11}}
+            {{/secret-link}}
+          </div>
+        {{/unless}}
+      {{#if (or (eq model.type "aws") (eq model.type "ssh") (eq model.type "pki"))}}
+        <div class="control">
+          <a href={{href-to
+              "vault.cluster.settings.configure-secret-backend"
+              model.id
+            }}
+            class="button has-icon-right is-ghost is-compact"
+            data-test-secret-backend-configure=true
+            >
+              Configure {{i-con glyph="chevron-right" size=11}}
+          </a>
+        </div>
+      {{/if}}
+    {{/p.levelRight}}
+  {{/page-header}}
+  {{#if options.tabs}}
+    <div class="box is-bottomless is-marginless is-fullwidth is-paddingless">
+      <nav class="tabs sub-nav">
+        <ul>
+          {{#each options.tabs as |oTab|}}
+            {{#if oTab.tab}}
+              {{#link-to 'vault.cluster.secrets.backend.list-root' (query-params tab=oTab.tab) tagName="li" activeClass="is-active" data-test-tab=oTab.label}}
```

Comment:
I kinda like the look of `'` in this helper value context. Single-quotes feel more like JS, which this is closer to, but it feels harder to maintain consistency. And indeed I'm seeing variations in this style below (https://github.com/hashicorp/vault/pull/4598/files#diff-489248d4f70417d5d6e1365d153fdb63R1). Just being super nitpicky so feel free to ignore. I'm mostly calling it out because I found it interesting. :)

Reply:
Ha, yeah this was just an inconsistency in a template I copied over - can't wait for prettier for templates!